src/style.css

```diff
@@ -128,6 +128,90 @@ footer a:hover,
   opacity: 0.7;
 }
 
+/* Modals */
+
+.centering {
+  justify-content: center !important;
+}
+
+.modal {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #333d;
+}
+
+.modal h4 {
+  margin-top: 20px;
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: white;
+  height: 90vh;
+  padding-bottom: 20px;
+  margin: 5vh 20vw;
+  border-radius: 15px;
+  overflow-y: scroll;
+}
+
+.img-frame {
+  position: relative;
+  background-color: #333;
```

Comment:
What do you think about changing the background color to a more transparent color? I believe this will make the image more visible and solve the low contrast issue of the `X` mark.

Reply:
Requested change completed.